demisto_sdk/commands/common/hook_validations/pack_unique_files.py

```diff
@@ -31,6 +31,7 @@
 SUPPORTED_CONTRIBUTORS_LIST = ['partner', 'developer']
 ISO_TIMESTAMP_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
 ALLOWED_CERTIFICATION_VALUES = ['certified', 'verified']
+SUPPORT_TYPES = ['partner', 'community', 'xsoar']
```

Comment:
I think it is better to use `PACK_SUPPORT_OPTIONS` in constants.py (and rm developer)